modules/react/common/spec/components.spec.tsx

```diff
@@ -173,29 +173,29 @@ describe('composeHooks', () => {
   });
 
   it('should merge properties from both hooks', () => {
-    const props = composeHooks(hook1, hook2)(model, {});
+    const props = composeHooks(hook1, hook2)(model, {}, null);
```

Comment:
New type errors showed the test didn't pass the 3rd parameter. Lack of type checking makes it so easy to miss these subtle changes.